### DISCUSSION TITLE
Preserve White Space and Line breaks option in template output

### DISCUSSION THREAD

**Discussion by blairliikala** [ExpressionEngine/ExpressionEngine#2994] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
There are some formats that need line breaks such as webvtt and caldav.  However, channel tags remove the trailing line breaks before the closing tag.

**Describe the solution you'd like**
A way to not not have line breaks, or any trim function, removed.  This could be implemented in a few ways:
- A template option in the admin, would be a global option for that template or a new template type.
- An entries parameter such as "trim=no" and would apply for that loop.
- A tag loop such as {no_trim} {/no_trim}

**Describe alternatives you've considered**
I use a custom field that always has an empty value right before the closing loop tag.
{other template code} [linebreak]
{blank_field}{/exp:channel:entries}

**Comment by robsonsobral** [comment]
A `trim` parameter could be interesting.

**Comment by intoeetive** [comment]
Adding a note here that I also saw the opposite request - ability to trim the unnecessary whitespaces. 

I do like an idea of per-template setting for this
